toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -409,6 +420,59 @@ os:
       permissions: "664"
 ```
 
+## dirConfig type
+
+Specifies options for placing a directory in the OS.
+
+Type is used by: [additionalDirs](#additionaldirs-dirconfig)
+
+<div id="dirconfig-path"></div>
+
+### sourcePath [string]
+
+The absolute path to the source directory that will be copied.
+
+### destinationPath [string]
+
+The absolute path in the target OS that the source directory will be copied to.
+
+Example:
+
+```yaml
+os:
+  additionalDirs:
+    - sourcePath: "home/files/targetDir"
+      destinationPath: "usr/project/targetDir"
+```
+
+### newDirPermissions [int]
```

Comment:
Please match the existing documentation for fileConfig.permissions.

The permission value is a string because at some point we may add support for the alternative formats that chmod supports.